src/app/[locale]/specs/lexicon/en.mdx

```diff
@@ -208,6 +231,34 @@ Note that unlike `object`, there is no `nullable` field on `params`.
 
 `params` can not be declared as a top-level named type in a schema `defs` array. This means that they can not be referenced by a `ref` or `union`.
 
+### `permission`
+
+This is a limited-scope type which is only ever used for the `permissions` field of the `permission-set` primary type, and should not appear in other schema contexts. They map to auth permissions.
+
+Type-specific fields:
+
+- `resource` (string, required): indicates what account resource is being accessed
+
+Additional `permission` fields depend on the resource type being declared. The set of resources may expand over time, but currently includes:
+
+- `repo`: public repository write permissions
+    - `collection` (array of strings, required): indicates record types. Must include at least one element (not empty array). Supports wildcard (`*`), but does not support partial-NSID glob patterns (`com.example.*`)
```

Comment:
I would add a note here:

Only collections under the same authority as the permission-set can be granted access to. This means that `*` here is basically the same as `<permission-set-group-nsid>.*`